snuba/clickhouse/query_dsl/accessors.py

```diff
@@ -109,7 +109,9 @@ def get_time_range_expressions(
                 Or(
                     [
                         String(OPERATOR_TO_FUNCTION[">="]),
+                        String(OPERATOR_TO_FUNCTION[">"]),
```

Comment:
Small functional change here.

Reply:
It's needed to correctly allow the time ranges to be calculated.

Reply:
yes, what was the issue?

Reply:
It's not an issue, it's a quality of life change. Now it's not necessary to have EXACTLY >=/< in order to pass the required time checks.